.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -1,8 +1,8 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-title: 'A very short description/title for the bug'
-labels: 🐞  Bug
+title: A very short description/title for the bug
+labels: "\U0001F41E  Bug"
```

Comment:
This does not really need to be change IMHO. Also, I'd keep the quotes so YAML will know that it's a string. We shouldn't confuse it, and tell it that just in case.